src/hosted-buttons/utils.js

```diff
@@ -289,41 +339,110 @@ export function getButtonColor(
   };
 
   return colorMap[color][fundingSource];
-}
-
-export function shouldRenderSDKButtons(
-  fundingSources: $ReadOnlyArray<FundingSources>
-): boolean {
-  return Boolean(fundingSources.length);
-}
+};
 
-export function appendButtonContainer({
+export const applyContainerStyles = ({
   flexDirection,
-  selector,
-}: BuildButtonContainerArgs) {
-  const elm = getElementFromSelector(selector);
+  buttonContainerId,
+}: ApplyButtonStylesProps): void | Error => {
```

Comment:
Does flow have checked exceptions? Not sure this return type is right.

Reply:
Good catch! I'll just return `void` here 😄